app/Controllers/Doctor/DoctorAuthController.cs

```diff
@@ -30,5 +35,13 @@ public async Task<IActionResult> AuthTest()
         {
             return Ok();
         }
+
+        // POST doctor/vaccination-slots
+        [HttpPost("vaccination-slots")]
+        [Authorize(AccountTypeEnum.Doctor)]
+        public async Task<IActionResult> AddVaccinationSlots([FromBody] NewVaccinationSlotRequest request)
```

Comment:
Czemu w tym kontrollerze daliscie ten endpoint? Kontroler to DoctorAuth a chyba dodawanie slotow do szczepien nie ma za duzo wspolnego z autoryzacja

Reply:
ok, zobacz teraz

Reply:
Git tylko nie usunąłeś tego serwisu z klasy DoctorAuthController